src/core/stream/index.ts

```diff
@@ -184,7 +188,7 @@ export default function Stream({
 
   // Creates pipelines for downloading segments.
   const segmentPipelinesManager = new SegmentPipelinesManager<any>(
-    transport, requestsInfos$, network$, warning$);
+    transportPipelineInfos.transportPipelines, requestsInfos$, network$, warning$);
```

Comment:
That'a a little difficult to read and understand: ``transportPipelineInfos.transportPipelines``

Why not just ``transport.pipelines`` and ``transport.options`` or something alike?